services/ir_pagerduty.rb

```diff
@@ -29,11 +29,11 @@ def attach_contact_methods_to_user(user)
   end
 
   def fetch_todays_schedules_users(sid)
-    today = Date.today
```

Comment:
Quite true, this would only happen if its ran on the crossover from one day to the other but we could just cache the date:

```ruby
private

def date_today
  @date_today ||= Date.today.to_s
end
```